test/Merging/abort.c

```diff
@@ -0,0 +1,714 @@
+// RUN: %llvmgcc -emit-llvm -g -c -o %t.bc %s
```

Comment:
What is the idea of this test case?
I see 700 new lines of code, but I have no glue what they are for.
Please add some description and add checks which you would like to use to validate your approach.

Reply:
The purpose of this test was to check that there would be no segfaults/crashes in case klee runs out of time and halts execution. However, if necessary, this can also be done more simply (e.g., a loop, ..). For now, we removed this version of the test.